src/LuaTransformer.ts

```diff
@@ -5514,6 +5516,42 @@ export class LuaTransformer {
                 case "WeakSet":
                     this.importLuaLibFeature(LuaLibFeature.WeakSet);
                     return;
+                case "Error":
```

Comment:
I don't really like this, could we move all error types to a const set of strings and just check `.has()`?